src/sentry/static/sentry/app/components/assistant/guideDrawer.jsx

```diff
@@ -0,0 +1,95 @@
+import Reflux from 'reflux';
+import PropTypes from 'prop-types';
+import React from 'react';
+import createReactClass from 'create-react-class';
+import ApiMixin from '../../mixins/apiMixin';
+import GuideStore from '../../stores/guideStore';
+import GuideActions from '../../actions/guideActions';
+
+// GuideDrawer is what slides up when the user clicks on a guide cue.
+const GuideDrawer = createReactClass({
+  displayName: 'GuideDrawer',
+
+  propTypes: {
+    guide: PropTypes.object.isRequired,
+    closeHandler: PropTypes.func.isRequired,
+  },
+
+  mixins: [ApiMixin, Reflux.listenTo(GuideStore, 'onGuideStateChange')],
+
+  getInitialState() {
+    return {
+      step: 0,
+    };
+  },
+
+  onGuideStateChange(data) {
+    this.setState({
+      step: data.currentStep,
+    });
+  },
+
+  nextHandler() {
+    GuideActions.nextStep();
+  },
+
+  usefulHandler(useful) {
+    this.api.request('/assistant/', {
```

Comment:
Are we supposed to use actions even for things that don't affect the store?

Reply:
Generally no, but I think api requests can be moved into an action creator (even though we're technically not firing any reflux actions). I only mentioned it in this case because you're already using actionCreators, otherwise I would leave it.

Reply:
Also action creators are not the same as actions. action creators in its most basic case just dispatch actions. Here's an example (and a more complicated case): https://github.com/getsentry/sentry/blob/master/src/sentry/static/sentry/app/actionCreators/plugins.jsx#L41